orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/JedisExecutionRepository.groovy

```diff
@@ -505,7 +511,7 @@ class JedisExecutionRepository implements ExecutionRepository {
       map.description = execution.description
     }
 
-    jedis.hdel(key, "config")
+    jedis.hdel(key, "config", "stageIndex")
```

Comment:
Looks like we were already cleaning up an old hash field here (`config` used to contain the serialized JSON) so I just added the new one